services/brig/src/Brig/Team/DB.hs

```diff
@@ -162,5 +163,8 @@ countInvitations t = fromMaybe 0 . fmap runIdentity <$>
     cqlSelect :: PrepQuery R (Identity TeamId) (Identity Int64)
     cqlSelect = "SELECT count(*) FROM team_invitation WHERE team = ?"
 
+-- | FUTUREWORK: two weeks after https://github.com/wearezeta/backend-issues/issues/775 has been
```

Comment:
Is this an accident, or is the rule going to be "backend-issues can be referred-to in code but not in issue comments"?

Reply:
accident.  :-)  will fix, too!